src/store/server/pageRendering.js

```diff
@@ -31,7 +31,7 @@ const getHead = () => {
     style: helmet.style.toString(),
     htmlAttributes: helmet.htmlAttributes.toString(),
     base: helmet.base.toString(),
-    noscript: helmet.noscript.toString()
+    noscript: helmet.noscript.toString(),
```

Comment:
please remove everywhere at the last position in an object the commas, thx.

Reply:
That's going to be a problem due to javascript is now default practice to add commas at end to reduce changes on a commit and are automated during lint process, an alternative I might add is keep a .prettierrc file with your preferred linting settings, you know if you like double quotes put it in, and when you commit changes run the prettier with default prettier options and when you work prettier with your liking...